From 8e64d6e5b68f568f3ac1cec23d52c62185d8e6dc Mon Sep 17 00:00:00 2001
From: Shaun Zhou <uranus-neptune@hotmail.com>
Date: Sat, 3 Jun 2023 09:55:34 +0800
Subject: [PATCH] Init k880g6 dts from ast2600-evb.dts

---
 arch/arm/dts/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 6c34b83336..42283f1ce2 100755
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -695,7 +695,7 @@ dtb-$(CONFIG_ARCH_ASPEED) += \
 	ast2600-slt.dtb \
 	ast2600-tacoma.dtb \
 	ast2600-intel.dtb \
-	ast2600-tacoma.dtb
+	ast2600-uboot-shaun-k880g6.dtb
 
 dtb-$(CONFIG_ARCH_STI) += stih410-b2260.dtb
 
-- 
2.34.1

